packages/gateway/src/graphql/schema.d.ts

```diff
@@ -588,6 +589,11 @@ export interface GQLRejectRegistrationInput {
   comment?: string
 }
 
+export interface GQLUpsertRegistrationIdentifierInput {
+  reason: string
+  comment?: string
+}
+
```

Comment:
Wonder how is this type generated? 🤔

Reply:
Isn't this autogenerated via our tooling?